packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -354,7 +354,7 @@ module.exports = {
 
     // enforce the location of single-line statements
     // http://eslint.org/docs/rules/nonblock-statement-body-position
-    'nonblock-statement-body-position': 'off',
+    'nonblock-statement-body-position': 'error',
```

Comment:
we generally prefer to be explicit about what setting we're using, even if it's the default - could we change this to specify `['error', 'beside', { overrides: {} }]`?